plugins/woocommerce-blocks/tests/e2e/tests/checkout/checkout-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -430,7 +430,89 @@ test.describe( 'Shopper → Checkout block → Place Order (guest user)', () =>
 	} );
 } );
 
-test.describe( 'Checkout Form Errors (guest user)', () => {
+test.describe( 'Shopper → Place Virtual Order', () => {
+	test.beforeEach( async ( { requestUtils } ) => {
+		await requestUtils.rest( {
+			method: 'PUT',
+			path: 'wc/v3/settings/general/woocommerce_ship_to_countries',
+			data: { value: 'disabled' },
+		} );
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.rest( {
+			method: 'PUT',
+			path: 'wc/v3/settings/general/woocommerce_ship_to_countries',
+			data: { value: 'all' },
+		} );
+	} );
+
+	test( 'can place a digital order when shipping is disabled', async ( {
+		checkoutPageObject,
+		frontendUtils,
+		localPickupUtils,
+		page,
+	} ) => {
+		await localPickupUtils.disableLocalPickup();
+
+		await frontendUtils.emptyCart();
+		await frontendUtils.goToShop();
+		await frontendUtils.addToCart( SIMPLE_PHYSICAL_PRODUCT_NAME );
+		await frontendUtils.goToCart();
+
+		await expect(
+			page.getByText( 'Shipping', { exact: true } )
+		).toBeHidden();
+
+		await frontendUtils.goToCheckout();
+
+		await expect(
+			page.getByText( 'Shipping', { exact: true } )
+		).toBeHidden();
+
+		await checkoutPageObject.fillInCheckoutWithTestData();
+		await checkoutPageObject.placeOrder();
+
+		await expect(
+			page.getByText( 'Thank you. Your order has been received.' )
+		).toBeVisible();
+
+		await localPickupUtils.enableLocalPickup();
```

Comment:
This seems out of place here. It's duplicated in the next test where it's more relevant

Reply:
By default, local pickup is enabled. I've added this part so that after running the test, the environment is reset to its default state. In the next test, I'm enabling local pickup before the test to ensure that it's enabled. For example, if the previous test would failed, it would leave the local pickup disabled. Does that make sense?

Reply:
Yep fair enough!